src/scripts/setup_test_data/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+ "1Dbywpp4w2zhXjZPgItrKvlh-raw.zip" : {
+   "recordid": "1Dbywpp4w2zhXjZPgItrKvlh",
+   "substudymemberships": "|mtb-alpha=NNJ18004|",
```

Comment:
Out of curiosity, why the pipes?

Reply:
No clue. I was replicating test data. This is a Bridge thing and i don't think it's used at all.